### ISSUE TITLE
projectUrl elements in nuspec now showing git:// prefixed url

### ISSUE THREAD

**Issue by mmitche** [dotnet/arcade#6962]
- [x] This issue is blocking 
- [ ] This issue is causing unreasonable pain

This blocks pushes to nuget.org. The projectUrl appears to have been changed over from https://dot.net, aspnetcore.net, etc. and is now showing a git:// prefixed cloning url instead. This blocks any nuget.org push because that is not viewed as a valid project url.

**Comment by mmitche** [comment]
This appears to only be dotnet/runtime packages, so it might be a change in runtime.

**Comment by leecow** [comment]
Hey Matt, a couple of things. Have we identified who to work with on the dotnet/runtime side? I thought we had some nuget validation baked into the pipeline (cc @adiaaida ). Is this not the case or is there a hole in the coverage?

cc @rbhanda

**Comment by adiaaida** [comment]
I don't think we have any validation that validates nuspecs. Just that each of the nuget packages exists either locally or remotely.

**Comment by mmitche** [comment]
The nuget validation worked fine. (nuget verify on a package works fine).

The work on the dotnet/runtime side is that the Microsoft.NETCore.App.* packages in runtime have the wrong project url. I don't think this is actually an arcade issue now since it was specific to just those packages. I'll transfer the issue over.
